pkg/translator/azure/property_names.go

```diff
@@ -0,0 +1,51 @@
+package azure
+
+var mappings = map[string]map[string]string{
+	"common": {},
+	"FrontDoorAccessLog": {
+		"trackingReference":   "",
+		"httpMethod":          "http.request.method",
+		"httpVersion":         "http.request.version",
+		"requestUri":          "url.full",
+		"hostName":            "server.address",
+		"requestBytes":        "http.request.body.size",
+		"responseBytes":       "http.response.body.size",
+		"userAgent":           "user_agent.original",
+		"clientIp":            "client.address",
+		"clientPort":          "client.port",
+		"socketIp":            "socket.address",
+		"timeTaken":           "http.server.request.duration",
+		"requestProtocol":     "",
+		"securityProtocol":    "",
+		"securityCipher":      "",
+		"securityCurves":      "",
+		"endpoint":            "",
+		"httpStatusCode":      "http.response.status_code",
+		"pop":                 "",
+		"cacheStatus":         "",
+		"matchedRulesSetName": "",
+		"routeName":           "http.route",
+		"referrer":            "http.request.header.referer",
+		"timeToFirstByte":     "",
+		"errorInfo":           "",
+		"originURL":           "",
+		"originIP":            "",
+		"originName":          "",
+		"result":              "",
+		"sni":                 "",
+	},
+}
+
+func ResourceLogKeyToSemConvKey(azName string, category string) (string, bool) {
+	if mapping := mappings[category]; mapping != nil {
```

Comment:
We can use the `if mapping, ok := mappings[category]; ok {` syntax here and below instead. Personally I prefer

```go
mapping, ok := mappings[category]
if ok {
...
}
```